src/main/java/mate/academy/springboot/swagger/util/ParamsParser.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class ParamsParser {
+    private static final int PROPERTY_INDEX = 0;
+    private static final int DIRECTION_INDEX = 1;
+    private static final String SEMICOLON = ";";
+    private static final String COLON = ":";
+
+    public static Sort getSortingParams(String sortingParams) {
```

Comment:
```suggestion
    public Sort getSortingParams(String sortingParams) {
```
make it a bean and autowire it